content/documentation/staging/installation-guide/index.adoc

```diff
@@ -102,6 +103,7 @@ To replace the existing rule and allow master access:
 gcloud compute firewall-rules update <firewall-rule-name> --allow <previous-ports>,tcp:8080,tcp:15000
 ----
 
+icon:bullhorn[size=1x]{nbsp} Istio deployments on private clusters also need extra ports to be opened. Check the https://istio.io/latest/docs/setup/platform-setup/gke/[Istio installation page for GKE,window="_blank"] to see all the extra installation steps for this platform.
```

Comment:
Good Monday @jshaughn ! Your English student has another delivery for you. Do you mind checking it out? :stuck_out_tongue_closed_eyes: Thank you!